contrib/buildahimage/Containerfile

```diff
@@ -67,8 +67,7 @@ ADD ./containers.conf /etc/containers/
 
 # Copy & modify the defaults to provide reference if runtime changes needed.
 # Changes here are required for running with fuse-overlay storage inside container.
```

Comment:
I think, but dont' know for sure, that this line still holds.  But wanted to double check.